build_tools/install_rocm_from_artifacts.py

```diff
@@ -321,6 +332,13 @@ def main(argv):
         action=argparse.BooleanOptionalAction,
     )
 
+    artifacts_group.add_argument(
+        "--miopen_plugin",
```

Comment:
```suggestion
        "--miopen-plugin",
```

it should convert to miopen_plugin in python though